src/main/java/dev/ebrydeu/spring_boot_library/domain/entities/Role.java

```diff
@@ -0,0 +1,43 @@
+package dev.ebrydeu.spring_boot_library.domain.entities;
+
+import jakarta.persistence.*;
+import lombok.*;
+import org.hibernate.proxy.HibernateProxy;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import java.util.Objects;
+
+@Entity
+@Table(name = "roles")
+@Builder
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@EntityListeners(AuditingEntityListener.class)
+public class Role extends Auditable{
```

Comment:
What point of Auditable in this ? Roles are constants which doesn't require any date or author stamps.